spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
I don't see that this is outside the scope of the bug fix. The bug fix is to make the tests pass in a way that make the tests tell the truth. Both solutions fix the bug but we want to merge in the one thats right.
Currently we have a failing test when run on its own. One fix gets the test to pass but continues to misrepresent what happens. The other fix actually fixes the root cause of the issue and doesn't require an unnecessary require.